shcheck.py

```diff
@@ -242,8 +256,9 @@ def main(options, targets):
             targets = f.read().splitlines()
         
 
-
+    json_out=[]
```

Comment:
I think this should be a dict, otherwise the JSON output is not JSON but a list

Reply:
You're right, I'm not certain the best route, you could encapsulate them in one dictionary, with an array that is {"results":[ ... ]}